bridges/otellogrus/go.mod

```
@@ -0,0 +1,22 @@
+module go.opentelemetry.io/contrib/bridges/otellogrus
+
+go 1.21
+
+require (
+	github.com/sirupsen/logrus v1.9.3
+	github.com/stretchr/testify v1.9.0
+	go.opentelemetry.io/otel/log v0.1.0-alpha.0.20240409140646-648b40eae158
+	go.opentelemetry.io/otel/sdk v1.24.0
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/go-logr/logr v1.4.1 // indirect
+	github.com/go-logr/stdr v1.2.2 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	go.opentelemetry.io/otel v1.25.0 // indirect
+	go.opentelemetry.io/otel/metric v1.25.0 // indirect
+	go.opentelemetry.io/otel/trace v1.25.0 // indirect
```

```suggestion
	go.opentelemetry.io/otel v1.26.0 // indirect
	go.opentelemetry.io/otel/metric v1.26.0 // indirect
	go.opentelemetry.io/otel/trace v1.26.0 // indirect
```

Should we use the latest version of otel?